pages/facts.html

```diff
@@ -1,12 +1,45 @@
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../stylesheets/styles.css">
+  <link rel="stylesheet" href="../stylesheets/facts.css">
   <title>Document</title>
 </head>
 <body>
-  
+  <header>
+    <h1 id="facts-title"><span class="smaller">Tiny</span> Creatures in <span class="smaller">Tiny</span> Hats</h1>
+    <nav>
+      <ul>
+        <li><button type="button" onclick="window.location.href='index.html'">Home</button></li>
+        <li><button type="button" onclick="window.location.href='gallery.html'">Gallery</button></li>
+        <li><button type="button" onclick="window.location.href='facts.html'">Facts</button></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <div class="flex-container">
+      <ul>
+        <li class="fact-item">
+          <img src="../images/CuteDoctor.jpeg" alt="Cute Doctor">
+          <span class="fact-text">
+            "Your level of depression decreases correlation to how long you stare at <span class="tiny">tiny</span> creatures in <span class="tiny">tiny</span> hats" 
+            <div class="number-of-doctors">-4 out of 5 doctors</div>
```

Comment:
`Div` elements cannot be children of `span` elements. This is causing an error in the HTML validator. See this Stack Overflow for more information: https://stackoverflow.com/questions/11314668/what-elements-can-a-span-tag-contain-in-html5